packit_service/worker/copr_build.py

```diff
@@ -157,11 +157,11 @@ def run_copr_build(self) -> HandlerResults:
             return HandlerResults(success=False, details={"msg": msg})
 
         except Exception as ex:
-            msg = f"There was an error while running a copr build. {msg_retrigger}"
-            self.project.pr_comment(self.event.pr_id, msg)
-            logger.error(f"error while running a copr build: {ex}")
-            msg = f"There was an error while running the build: {ex}"
-            r.report("failure", msg, check_name=check_name)
+            msg = f"There was an error while running a copr build: {ex}"
+            logger.error(msg)
+            self.project.pr_comment(self.event.pr_id, f"{msg}\n{msg_retrigger}")
+            # status text can't be too long, put just the exception there
+            r.report("failure", f"{ex}", check_name=check_name)
```

Comment:
I would put here a static text saying that the build failed and they should check the pr comment

I can easily imagine how some exception messages could make this really cryptic, e. g. keyerror

Reply:
You thumbs-upped my [comment](https://github.com/packit-service/packit-service/issues/105#issuecomment-526092170) about putting just the exception there :-)

But I see your point, what about now?

Reply:
I thumbs up everything.

Reply:
go thumb up yourself  :+1: